### PR TITLE
en: Add monitor multiplecluster and thanos doc

### PR DESCRIPTION
**Pull request by qiancai**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1254
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -290,3 +289,57 @@ The default Prometheus and alert configuration do not support sending alert mess
...
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+For a complete configuration example, refer to [Example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-multiple-cluster-non-tls) in the TiDB Operator repository.
+
+### Monitor multiple clusters using Grafana
+
+If the `tidb-monitor-initializer` image is earlier than v4.0.14 or v5.0.3, to monitor multiple clusters, you can take the following steps in each Grafana Dashboard:
```
Comment (Liuxiaozhen12) [inline en/monitor-a-tidb-cluster.md:339]: `is earlier than v4.0.14 or v5.0.3,`why we have to mention two versions? I think v4.0.14 is earlier than v5.0.3 so v5.0.3 seems to be unnecessary.

**Review by Liuxiaozhen12 — APPROVED** [review]
rest LGTM.

File: en/monitor-a-tidb-cluster.md (hunk 1 above)
Comment (qiancai) [reply en/monitor-a-tidb-cluster.md:339]: It means versions such as v4.0.13 and v5.0.2.  @DanielZhangQD please help check if my understanding is correct.

Comment (TomShawn) [reply en/monitor-a-tidb-cluster.md:339]: > `is earlier than v4.0.14 or v5.0.3,`why we have to mention two versions? I think v4.0.14 is earlier than v5.0.3 so v5.0.3 seems to be unnecessary.

v4.0.14 is actually later than v5.0.3.

**Pull request merged**